src/time_integration/paired_explicit_runge_kutta/methods_PERK2.jl

```diff
@@ -118,6 +117,7 @@ end
     - `base_path_monomial_coeffs` (`AbstractString`): Path to a file containing 
       monomial coefficients of the stability polynomial of PERK method.
       The coefficients should be stored in a text file at `joinpath(base_path_monomial_coeffs, "gamma_$(num_stages).txt")` and separated by line breaks.
+    - `dt_opt` (`Float64`): Optimal time step size for the simulation.
```

Comment:
```suggestion
    - `dt_opt` (`Float64`): Optimal time step size for the simulation setup.
```

Maybe be a little more distinctive here ?